nilearn/plotting/surface/surf_plotting.py

```diff
@@ -396,22 +396,22 @@ def plot_surf_stat_map(
     hemi=DEFAULT_HEMI,
     view=None,
     engine=DEFAULT_ENGINE,
+    cmap=DEFAULT_DIVERGING_CMAP,
+    colorbar=True,
+    avg_method=None,
     threshold=None,
     alpha=None,
+    bg_on_data=False,
+    darkness=0.7,
     vmin=None,
     vmax=None,
-    cmap=DEFAULT_DIVERGING_CMAP,
-    colorbar=True,
     symmetric_cbar="auto",
     cbar_tick_format="auto",
-    bg_on_data=False,
-    darkness=0.7,
     title=None,
-    title_font_size=18,
+    title_font_size=None,
```

Comment:
Here I've just changed the order of parameters to be consistent with the previous functions.